assets/css/structure.css

```diff
@@ -0,0 +1,42 @@
+body {
+  font-family: var(--primary-font);
+}
+
+.section {
+  border: var(--border);
+  margin: var(--Large);
+  padding: var(--Large);
+  background-color: var(--background-color);
+}
+
+.quote {
+  font-weight: var(--normal-font);
+  margin-bottom: var(--Large);
+  color: var(--primary-color);
+}
+
+.author {
+  font-weight: var(--bold-font);
+  text-align: right;
+  margin-bottom: var(--Large);
+  color: var(--primary-color);
+}
+
+/*buttons*/
+
+.buttons {
+  padding-top: var(--Large);
+}
+
+.wisdom-button {
+  font-weight: var(--thin-font);
+  margin: var(--small);
+  padding: var(--xSmall);
+  color: var(--secondery-color);
+}
+
+.famous-qoutes-button {
```

Comment:
This is almost identical to the above button. The only difference is margin which I'm not sure what's the reason. First, I guess, tags are more links than buttons. So I recommend a tag instead. Second, if we have a same design for these tags/buttons. I suggest to use just one class for all.

Reply:
Fixed.

Reply:
Fixed.